Makefile

```diff
@@ -212,7 +212,29 @@ lint: build install-linters gopath
 	@rm -rf ${LOCAL_GOPATH}/src/${GO_PACKAGE_PREFIX}/vendor
 	@cp -af vendor/* ${LOCAL_GOPATH}/src/
 	@go build -race github.com/clearlinux/clr-installer/...
-	@gometalinter.v2 --deadline=10m --tests --vendor \
+	@gometalinter.v2 --deadline=10m --tests --vendor --skip=vendor \
+	--exclude=vendor --disable-all \
```

Comment:
Do we need this `--exclude=vendor --disable-all \` if we are already skipping vendor `--skip=vendor`?

Reply:
exlcude Exclude messages matching these regular expressions
but skip, Skip directories with this name when expanding '...'.

So the skip should shorten the processing time by never running on the vendor code .